rotkehlchen/tests/api/test_assets.py

```diff
@@ -919,6 +919,63 @@ def test_search_assets_with_levenshtein(rotkehlchen_api_server):
     )
     assert_error_response(response, contained_in_msg='Failed to deserialize ChainID value near')
 
+    with rotkehlchen_api_server.rest_api.rotkehlchen.data.db.user_write() as cursor:
```

Comment:
you are adding to a huge test. Please don't do that. You are testing a completely different functionality. The test started 400 lines above and you are adding another 100 to it with this.

Split it out.